source/fundamentals/indexes.txt

```diff
@@ -371,3 +384,97 @@ The following example creates a unique, descending index on the ``theaterId`` fi
 
 Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB server manual for more information.
 .. driver-content-end
+
+Remove an Index
+---------------
+
+You can remove any unused index that isn't a :ref:`unique index
+<unique-indexes>`. 
+
+The following sections show the three ways to remove indexes:
+
+- Using an index specification document
+- Using an indexed name field
+- Removing all indexes
+
+Index Specification Document
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass the index specification documenent to the ``dropIndex()`` method to
+remove an index from a collection. 
+
+The following snippet removes an ascending index on the ``title`` field
+in a collection:
+
+.. code-block:: java
+
+   collection.dropIndex(Indexes.ascending("title"));
+
+.. important::
+
+   This does not work for a text index. To remove a text index, see the
+   :ref:`Name Field <name_field>` section.
+
+.. _name_field:
+
+Name Field
+~~~~~~~~~~
+
+Pass the indexed ``name`` field to the ``dropIndex()`` method to remove
+an index from a collection. You must do this to remove a text index.
```

Comment:
Since you've mentioned that you need to do this for a text index in the prior section, is it necessary to mention here? Doesn't removing an index by the name work for any type of index? And if so, couldn't it be confusing to call out text index?

Reply:
I'll remove the mention here since it does work for all indexes.